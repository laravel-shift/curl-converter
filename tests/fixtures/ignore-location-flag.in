curl --location -L https://example.com
