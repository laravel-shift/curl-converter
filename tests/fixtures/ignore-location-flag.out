Http::get('https://example.com');
